Microsoft.DotNet.Interactive/IExtensibleKernel.cs

```diff
@@ -2,13 +2,12 @@
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
 using MLS.Agent.Tools;
-using System.IO;
 using System.Threading.Tasks;
 
 namespace Microsoft.DotNet.Interactive
 {
     public interface IExtensibleKernel
     {
-        public Task LoadExtensionsInDirectory(IDirectoryAccessor directory, KernelInvocationContext invocationContext);
+        Task LoadExtensionsInDirectory(IDirectoryAccessor directory, KernelInvocationContext invocationContext);
```

Comment:
Could we rename to `LoadExtensionsFromDirectory`?

Reply:
done!